include/cvc5/cvc5_proof_rule.h

```diff
@@ -356,7 +356,17 @@ enum ENUM(ProofRule) : uint32_t
    * SAT solver. \endverbatim
    */
   EVALUE(SAT_REFUTATION),
-
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **DRAT Refutation**
+   *
+   * .. math::
+   *   \inferrule{F_1 \dots F_n \mid D, P}{\bot}
+   *
+   * where :math:`F_1 \dots F_n` correspond to the input clauses in the
+   * DIMACS file `D` and `P` is a DRAT proof. \endverbatim
```

Comment:
Define this more generally. Refer to clauses rather than input clauses. Also, clarify what is the structure of `P`.